package.json

```
@@ -6,6 +6,9 @@
   "scripts": {
     "test": "jest",
     "dev": "node ./src/index.js",
+    "daemon": "yarn daemon:run",
+    "daemon:run": "yarn build && pm2 start ./src/index.js --name game-checkers-bot --log './logs/current.log'",

```

There is not build command.
```suggestion
    "daemon:run": "pm2 start ./src/index.js --name game-checkers-bot --log './logs/current.log'",

```

---

Because it's plain JS we don't use here any builder/bundler.